sdk/nodejs/runtime/invoke.ts

```diff
@@ -349,27 +302,6 @@ async function invokeAsync(
     }
 }
 
-/**
- * {@link StreamInvokeResponse} represents a (potentially infinite) streaming
- * response to `streamInvoke`, with facilities to gracefully cancel and clean up
- * the stream.
- */
-export class StreamInvokeResponse<T> implements AsyncIterable<T> {
```

Comment:
Remove the dependency on `@pulumi/query`?

Reply:
Good catch, will clean that all up as well